lib/runner.js

```diff
@@ -265,10 +265,17 @@ Runner.prototype.fail = function(test, err) {
  * @param {Error} err
  */
 Runner.prototype.failHook = function(hook, err) {
+  hook.originalTitle = hook.originalTitle || hook.title;
   if (hook.ctx && hook.ctx.currentTest) {
-    hook.originalTitle = hook.originalTitle || hook.title;
     hook.title =
       hook.originalTitle + ' for "' + hook.ctx.currentTest.title + '"';
+  } else {
+    var parent = hook.parent.title
+      ? hook.parent.title
+      : hook.parent.root
+      ? '{root}'
+      : '';
+    hook.title = hook.originalTitle + ' in "' + parent + '"';
```

Comment:
Venturing into ternary op abuse...
```js
    var parent = hook.parent.title
      ? hook.parent.title
      : hook.parent.root
        ? '{root}'
        : '';
```

I indented the latter -- if prettier did this, it's borken...

Reply:
I had to use `// prettier-ignore`

Reply:
Chris won't go for that (turning off prettier).
By reformulate, I meant use two variables or `if` logic -- combining two (or more) ternary operators on a single variable is too much added complexity (metrics-wise) for a single variable (IMO).

Reply:
guilty as charged. 

the idea with prettier is to just let it do whatever it's going to do.  sometimes it might format something a way you don't like, or I don't like, or maybe *nobody* likes, but the intent is to make formatting a non-issue by complete, final, *mindless* adherence to whatever it does.  when we pray at the unholy altar of prettier, it demands no less than total submission.

We *do* have agency when using nested ternaries though.  I try to avoid them in my own code.  but if I've requested people eliminate them in the past, I think I'm over it now.  there are more important things to worry about.